tests/odov3/odo_test.go

```diff
@@ -61,30 +55,28 @@ func TestOdo(t *testing.T) {
 	// more at: https://onsi.github.io/ginkgo/#dynamically-generating-specs
 	g := NewGomegaWithT(t)
 
-	// make stacks path absolute, it is required because later we are using it from the random directory.
-	stacksDir, err := filepath.Abs(stacksDir)
-	g.Expect(err).To(BeNil())
+	files := make([]string, 0)
 
-	files, err := ioutil.ReadDir(stacksDir)
-	g.Expect(err).To(BeNil())
+	if filesStr != "" {
```

Comment:
No, this is not possible otherwise the PR will test all stacks when there haven't been any devfile changes. I was also going to originally do this as well.

Reply:
Hmm? Why's that, aren't we passing the changed stacks with the `--filesStr` argument?